### DISCUSSION TITLE
Billing usage of function usage shows weird numbers and overflowing data

### DISCUSSION THREAD

**Discussion by nazmifeeroz** [vercel/community#2687] [Help]
### Description

<img width="1718" alt="image" src="https://github.com/vercel/vercel/assets/26022852/1c92c8f2-7a89-4891-a095-3b5b74b2f846">

I'm on Pro trial plan. I'm seeing this `0.81 GB-Hrs / 0 GB-Hrs`

This don't make sense? is it cause i'm on trial plan?

**Comment by swarnava** [comment]
Hi, we are investigating this currently.
